kotest-property/src/jvmTest/kotlin/com/sksamuel/kotest/property/NullableTest.kt

```diff
@@ -0,0 +1,57 @@
+package com.sksamuel.kotest.property
+
+import io.kotest.core.spec.style.FunSpec
+import io.kotest.matchers.ints.shouldBeBetween
+import io.kotest.matchers.shouldBe
+import io.kotest.matchers.shouldNotBe
+import io.kotest.property.checkAll
+import io.kotest.property.forAll
+import io.kotest.property.forNone
+import io.kotest.property.withAssumptions
+
+class NullableTest : FunSpec({
+   test("forAll with implicit nullable arbitaries") {
+      val iterations = 1000
+      val classifications = forAll<Int?>(iterations) { num ->
+         classify(num == null, "null", "non-null")
+         true
+      }.classifications()
+      classifications["null"]?.shouldBeBetween(300, 600)
+      classifications["non-null"]?.shouldBeBetween(300, 600)
+   }
+
+   test("forNone with implicit nullable arbitraries") {
+      val iterations = 1000
+      val classifications = forNone<Int?>(iterations) { num ->
+         classify(num == null, "null", "non-null")
+         false
+      }.classifications()
+      classifications["null"]?.shouldBeBetween(300, 600)
+      classifications["non-null"]?.shouldBeBetween(300, 600)
+   }
+
+   test("checkAll with implicit nullable arbitraries") {
+      val iterations = 1000
+      val classifications = checkAll<Int?>(iterations) { num ->
```

Comment:
If you specify a seed then you can set the values precisely.

Reply:
@sksamuel I added seed !